esx_service/utils/auth.py

```diff
@@ -1,4 +1,199 @@
 import random
+import logging
+import auth_data
+import os
 
+auth_mgr = None
+def connect_auth_db():
+    global auth_mgr
+    if not auth_mgr:
+        db_path = os.path.join(auth_data.get_auth_db_path())
```

Comment:
Since `auth_data.get_auth_db_path()` takes no arguments it can be completely encapsulated inside `auth_data.AuthorizationDataManager`. The constructor for `AuthorizationDataManager` would then not require any arguments either. The path would just be set in the constructor by calling `get_auth_db_path` directly.